plugins/MauticSocialBundle/Translations/en_US/messages.ini

```diff
@@ -100,7 +100,7 @@ mautic.mauticSocial.permissions.header="Social Permissions"
 mautic.mauticSocial.permissions.monitoring="Monitoring - User has access to"
 mautic.mauticSocial.permissions.tweet="Tweet - User has access to"
 mautic.mautic.social.monitoring.form.confirmdelete="Delete monitor?"
-mautic.mautic.integration.Twitter.form.confirmdelete="Delete tweet?"
+mautic.integration.Twitter.form.confirmdelete="Delete tweet?"
```

Comment:
Why was this changed? Now all the language translations will have to translate this again.

Reply:
https://github.com/mautic/mautic/pull/11944#issuecomment-1426785218

This was the only way I was able to find to get the modal dialog to display. I could be missing something, in which case please let me know.

Reply:
You are right. It must heave been broken from the beginning. Thanks for the info!